### PR TITLE
Added

### PR DESCRIPTION
**Pull request by abhaygzb15**

### CODE REVIEW

**Comment by ayushsarode** [comment]
Please remove line 43 and insert the following HTML code:
```html
<h5 class="Role">Contributor</h5>
```

File: index.html [hunk 1]
```diff
@@ -36,10 +36,19 @@ <h5 class="Main">Maintainter</h5>
...
     <!-- Please place the code below this line -->
-
-
-    
-    
+    <figure class="Card">
+        <div class="profile-image"><img src="https://github.com/github.png?size=460" alt="Profile-Image" /></div>
+        <figcaption>
+            <h3>Abhay Pawar</h3>
+            <h5 class="Main Role">Contributor</h5>
```
Comment (ayushsarode) [inline index.html:43]: remove line 43 and add 

```html
<h5 class="Role">Contributor</h5>
```

**Comment by ayushsarode** [comment]
Hey @abhaygzb15 , would you like me to merge the pull request right away? Keep in mind that if I merge it now, it won't count towards your Hacktoberfest contributions since Hacktoberfest officially starts on October 1st.
If you're flexible, I'll merge it after 1st October to count in.

**Comment by abhaygzb15** [comment]
Yeah actually i forgot it..Thanks @ayushsarode for reminding me..Yes please do it after 1st october

**Comment by ayushsarode** [comment]
> Yeah actually i forgot it..Thanks @ayushsarode for reminding me..Yes please do it after 1st october

Sure! 
I'll merge it after 1 Oct.

**Comment by ayushsarode** [comment]
_Thank you for your valuable contribution!_

**Pull request merged**